### ISSUE TITLE
[Question] lib.dynamicContent: How to fetch only certain fields?

### ISSUE THREAD

**Issue by vigri** [benjaminkott/bootstrap_package#945]
# Support

## Description

Hello,

I would like to ask if it's possible to fetch only certian fields with `lib.dynamicContent`

Something like `<f:cObject typoscriptObjectPath="lib.dynamicContent" data="{colPos: '1', field: 'subheader'}" />`

This is the relevant part inside my fluid template
```
<h1 class="p-1">XXX HEADER XXX</h1>
<p class="lead p-1 font-weight-bold">XXX SUBHEADER XXX</p>
```

I've tried to modify `lib.dynamicContent` and add for instance `lib.dynamicContent.20.select.selectFields = header`

This unfortunately results in an error: `ERROR: Content Element with uid "5" and type "" has no rendering definition!`

Thanks in advance and best regards

**Comment by gilbertsoft** [comment]
Hey @vigri, what exactly are you trying to achieve?

**Comment by vigri** [comment]
Hey @gilbertsoft,

I'm using `lib.dynamicContent` in my own project.
On my root page I've a content element of type "only header" where the header and subheader are filled out.

Both parts should now be inserted in my fluid template.

If I only use `<f:cObject typoscriptObjectPath="lib.dynamicContent" data="{colPos: '1'}" />` I've no control about the individual elements (header / subheader). Therefore I would like to "split" them and get rid of the standard wrapping.

I hope my intention is clearer now, even if I think that it is not the best practice.

Thank you and best regards

**Comment by gilbertsoft** [comment]
Yeah, that's I was guessing. Did you create the CE by your own? The default header element does this already see images.

![grafik](https://user-images.githubusercontent.com/25326036/98439424-19aee980-20f2-11eb-87c5-f0a9adc1c18d.png)
![grafik](https://user-images.githubusercontent.com/25326036/98439442-30edd700-20f2-11eb-9236-b83e055c1add.png)
